### PR TITLE
SAPP-2500 route promotion label

### PR DESCRIPTION
**Pull request by singularo**
Add labels and annotations to the generated/test code.
Add support for labels on routes to Shepherd.

### CODE REVIEW

File: DrupalContentGenerate.php [hunk 1]
```diff
@@ -64,6 +64,18 @@
     'vid'                   => 'shp_environment_types',
     'name'                  => 'Dev',
     'field_shp_base_domain' => $domain_name,
+    'field_shp_annotations' => [
+      [
+        'key' => 'haproxy.router.openshift.io/ip_whitelist',
```
Comment (pingers) [inline DrupalContentGenerate.php:69]: We should leave this out - not going to be used in 4.x anyway right?

Comment (singularo) [reply DrupalContentGenerate.php:69]: Presume we dont need it, I reckon leave it in as an example, but commented.

Comment (singularo) [reply DrupalContentGenerate.php:69]: Presume we dont need it, I reckon leave it in as an example, but commented.

**Comment by pingers** [comment]
:ship:

**Pull request merged**